projects/exui-common-lib/src/lib/components/search-location/search-location.component.ts

```diff
@@ -86,6 +86,6 @@ export class SearchLocationComponent implements OnInit {
   }
 
   public getDisplayName(selectedLocation: LocationModel): string {
-    return selectedLocation.site_name;
+    return selectedLocation.court_name;
```

Comment:
do we need a separate method to get the court_name, can it be put in html directly?